PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -174,6 +174,58 @@ def miniAOD_customizeCommon(process):
     for idmod in electron_ids:
         setupAllVIDIdsInModule(process,idmod,setupVIDElectronSelection)
 
+    # Adding puppi jets
+    process.load('CommonTools.PileupAlgos.Puppi_cff')
+    process.patCandidates += process.puppi
```

Comment:
we are running unscheduled... what is this for?
btw I do not see a keep statement for slimmedJetsPuppi, how are jets store?